modules/javafx.graphics/src/main/native-glass/mac/GlassMenu.m

```diff
@@ -385,6 +408,15 @@ - (void)_setPixels:(jobject)pixels
     jMenuClosedMethod  = (*env)->GetMethodID(env, jMenuClass, "notifyMenuClosed",  "()V");
     if ((*env)->ExceptionCheck(env)) return;
     jDelegateMenuField = (*env)->GetFieldID(env,  jMenuDelegateClass, "menu", "Lcom/sun/glass/ui/Menu;");
+    if ((*env)->ExceptionCheck(env)) return;
+    jPixelsWidthField = (*env)->GetFieldID(env, pixelsClass, "width", "I");
+    if ((*env)->ExceptionCheck(env)) return;
+    jPixelsHeightField = (*env)->GetFieldID(env, pixelsClass, "height", "I");
+    if ((*env)->ExceptionCheck(env)) return;
+    jPixelsScaleXField = (*env)->GetFieldID(env, pixelsClass, "scalex", "F");
+    if ((*env)->ExceptionCheck(env)) return;
+    jPixelsScaleYField = (*env)->GetFieldID(env, pixelsClass, "scaley", "F");
+    if ((*env)->ExceptionCheck(env)) return;
```

Comment:
The last check is not needed, you are going to return either there is a pending exception or not.